xclim/sdba/properties.py

```diff
@@ -115,30 +160,33 @@ def var(da: xr.DataArray, *, group: str | Grouper = "time") -> xr.DataArray:
 
     Returns
     -------
-    xr.DataArray
+    xr.DataArray, [square of the input units]
       Variance of the variable.
 
     Examples
     --------
     >>> pr = open_dataset(path_to_pr_file).pr
     >>> var(da=pr, group="time.season")
     """
-    attrs = da.attrs
+    units = da.units
     if group.prop != "group":
         da = da.groupby(group.name)
     out = da.var(dim=group.dim)
-    out.attrs.update(attrs)
-    out.attrs["long_name"] = "Variance"
-    u = xc.core.units.units2pint(attrs["units"])
-    u2 = u**2
+    u2 = xc.core.units.units2pint(units) ** 2
     out.attrs["units"] = xc.core.units.pint2cfunits(u2)
-    out.name = "variance"
     return out
 
 
-@update_xclim_history
-@register_statistical_properties(aspect="marginal", seasonal=True, annual=True)
-def skewness(da: xr.DataArray, *, group: str | Grouper = "time") -> xr.DataArray:
+var = StatisticalProperty(
+    identifier="var",
+    aspect="marginal",
+    cell_methods="time: var",
+    compute=_var,
+    measure="xclim.sdba.measures.RATIO",
```

Comment:
```suggestion
    measure="xclim.sdba.measures.ratio",
```

`eval(xclim.sdba.properties.var.measure)` fails with `AttributeError: module 'xclim.sdba.measures' has no attribute 'RATIO'`. Mais, ça fonctionne en minuscule.

Reply:
nevermind, je n'avais pas vu get_measure()